### PR TITLE
Added sorting algorithm in articles route if param[:sorted] is defined

### PR DESCRIPTION
**Pull request by jerry1ye10**

### CODE REVIEW

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -9,6 +9,9 @@ def index
     else
       @articles = Article.all
     end
+    if params[:sorted]
```
Comment (NicholasLYang) [inline app/controllers/articles_controller.rb]: sorted is not a great param. I'd do order_by with different options, Right now it's sorta hard to figure out what you're sorting by. Also, consider using order, cause right now afaik, sort_by sorts in Ruby, while order should sort in SQL, which is far far more efficient

Comment (NicholasLYang) [reply app/controllers/articles_controller.rb]: so like order_by=date, order_by=title, etc.

Comment (NicholasLYang) [reply app/controllers/articles_controller.rb]: https://stackoverflow.com/questions/14897239/are-the-order-method-parameters-in-activerecord-sanitized-by-default

Comment (jkao1) [reply app/controllers/articles_controller.rb]: Only sort by rank and date for now.

**Comment by jkao1** [comment]
minimal orders are here. merged

**Pull request merged**